### PR TITLE
add cli export

### PR DESCRIPTION
**Pull request by tcitworld**
- [x] One more test for export file path
- [ ] Little documentation

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | WIP
| Translation   | no
| Fixed tickets | 
| License       | MIT

### CODE REVIEW

File: src/Wallabag/CoreBundle/Command/ExportCommand.php [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+            ->getBuilderForAllByUser($user->getId())
+            ->getQuery()
+            ->getResult();
+
+        $output->write(sprintf('Exporting %d entrie(s) for user « <comment>%s</comment> »... ', count($entries), $user->getUserName()));
+
+        $filePath = $input->getArgument('filepath');
+        if (!$filePath) {
+            $filePath = $this->getContainer()->getParameter('kernel.root_dir') . '/../' . sprintf('%s-export', $user->getUsername());
+        }
```
Comment (nicosomb) [inline src/Wallabag/CoreBundle/Command/ExportCommand.php]: maybe you can add lines to separate blocks here.

Comment (nicosomb) [inline src/Wallabag/CoreBundle/Command/ExportCommand.php]: I don't if it's a good thing to have export filenames based on username (even if I know that these files are not available from the web server).

Comment (tcitworld) [inline src/Wallabag/CoreBundle/Command/ExportCommand.php]: Well it's better in order to find them.

**Comment by tcitworld** [comment]
No idea what the hell `php-cs-fixer` does here.

File: src/Wallabag/CoreBundle/Command/ExportCommand.php (hunk 1 above)
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Command/ExportCommand.php]: I'm ok with that.

File: src/Wallabag/CoreBundle/Command/ExportCommand.php [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    }
+
+    /**
+     * Fetches a user from its username.
+     *
+     * @param string $username
+     *
+     * @return \Wallabag\UserBundle\Entity\User
+     */
+    private function getUser($username)
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Command/ExportCommand.php]: You don't really need a dedicated function when you only use that function once.

File: tests/Wallabag/CoreBundle/Command/ExportCommandTest.php [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+        $command = $application->find('wallabag:export');
+
+        $tester = new CommandTester($command);
+        $tester->execute([
+            'command' => $command->getName(),
+            'username' => 'admin',
+        ]);
+
+        $this->assertContains('Exporting 6 entrie(s) for user « admin »... Done', $tester->getDisplay());
```
Comment (j0k3r) [inline tests/Wallabag/CoreBundle/Command/ExportCommandTest.php]: Maybe you can add an extra test on the file (like you did for the test below)

File: src/Wallabag/CoreBundle/Command/ExportCommand.php (hunk 1 above)
Comment (nicosomb) [inline src/Wallabag/CoreBundle/Command/ExportCommand.php]: OK with the username in the filename?

Comment (j0k3r) [inline src/Wallabag/CoreBundle/Command/ExportCommand.php]: Yep. It is sluggified. So it should be safe.

**Comment by tcitworld** [comment]
Will update doc when it will be migrated (need a proper admin documentation, separated from user docs).

**Pull request merged**